spec/syntax.md

```diff
@@ -357,19 +357,19 @@ Function calls do not require an operand as an argument,
 but an _expression_ must not be completely empty.
 
 ```abnf
-expression = "{" [s] (((literal / variable) [s annotation]) / annotation) [s] "}"
+expression = "{" [s] (((literal / argtoken / variable) [s annotation]) / annotation) [s] "}"
 annotation = function *(s option)
 option = name [s] "=" [s] (literal / nmtoken / variable)
 ```
 
 Expression examples:
 
 ```
-{|1.23|}
+{1.23}
```

Comment:
we shouldn't remove the quoted examples, since they are still valid. We can augment the examples with unquoted ones. It would be useful to add or change the quoted ones to be something where the quotes add value, e.g. perhaps more like:

```
{1.23}
{|1.23|}
{1.23 :number maxFractionDigits=1}
{|-1.23| :number maxFractionDigits=2}
{|My Brand Name| :linkify href=|foobar.com|}
```

Reply:
I added a couple such examples.